llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -239,6 +239,19 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
   }
   case TargetOpcode::G_SEXT_INREG:
     return selectSExtInreg(MI, MIB);
+  case TargetOpcode::G_FRAME_INDEX: {
+    // TODO: We may want to replace this code with the SelectionDAG patterns,
+    // which fail to get imported because it uses FrameAddrRegImm, which is a
+    // ComplexPattern
+    Register DstReg = MI.getOperand(0).getReg();
+
+    if (!MRI.getType(DstReg).isPointer())
```

Comment:
Do we need this check? AArch64 has it but they were primarily checking for address space 0. The destination should always be some sort of pointer right?

Reply:
You're right, let me remove this